code/controllers/subsystem/mapping.dm

```diff
@@ -196,7 +196,6 @@ SUBSYSTEM_DEF(mapping)
 	var/list/space_ruins = levels_by_trait(ZTRAIT_SPACE_RUINS)
 	if (space_ruins.len)
 		seedRuins(space_ruins, CONFIG_GET(number/space_budget), list(/area/space), space_ruins_templates)
-	seedStation()
```

Comment:
why is this removed?

Reply:
why did you remove yourself from my life